bin/load-test.js

```diff
@@ -0,0 +1,200 @@
+var debug         = require('debug')('queue:bin:load-test');
+var base          = require('taskcluster-base');
+var Promise       = require('promise');
+var _             = require('lodash');
+var v1            = require('../routes/api/v1');
+var https         = require('https');
+var http          = require('http');
+var hawk          = require('hawk');
+var taskcluster   = require('taskcluster-client');
+var assert        = require('assert');
+
+/** Launch server */
+var launch = async function(profile) {
+  debug("Launching with profile: %s", profile);
+
+  // Load configuration
+  var cfg = base.config({
```

Comment:
Yes... config loading could be MUCH smarter... I hate how stupid it is... :)
It's really a foot gun and hard to get new people started, I moved my hack from f? to r?, and since you showed interest I figure you might like to review it:
https://bugzilla.mozilla.org/show_bug.cgi?id=1084661#c13

I haven't really seen much interest from others, and I can't really tell if the idea here (abusing YAML custom syntax) is the right or wrong amount of crazy :)